code/game/gamemodes/cult/ritual.dm

```diff
@@ -194,10 +194,9 @@
 	return 1
 
 /obj/item/weapon/tome/proc/finale_runes_ok(mob/living/user, obj/effect/rune/rune_to_scribe)
-
-	if(ticker.mode.name == "cult")
+	var/datum/game_mode/cult/cult_mode = ticker.mode
+	if(GAMEMODE_IS_CULT)
 		if(!canbypass == 1)//not an admin-tome, check things

```

Comment:
Yes, but you don't need to do `!canbypass == 1`. It just needs to be `!canbypass`.